README.md

```diff
@@ -221,14 +183,54 @@ Example data:
   "UUID": "b8b624e4-ec68-4e08-b5db-dfdc9ec84fea"
 }
 ```
-## Validation Messages
+
+### Error file schema
+The output of a metadata file validation should be a JSON file conforming to the [error file schema](errors/errorFileSchema.json)  
+An example of an error file conforming to this schema.
+```json
+{
+  "consignmentId" : "5049c395-6124-40fd-bffa-3fe44223bbd0",
+  "date" : "2025-01-22",
+  "fileError" : "SCHEMA_VALIDATION",
+  "validationErrors" : [
+    {
+      "assetId" : "test/hi.txt",
+      "errors" : [
+        {
+          "validationProcess" : "SCHEMA_CLOSURE_CLOSED",
+          "property" : "Closure Start Date",
+          "errorKey" : "type",
+          "message" : "Must be provided for a closed record"
+        }
+        
+      ],
+      "data" : [
+        {
+          "name" : "Closure Start Date",
+          "value" : ""
+        }
+      ]
+    }
+  ]
+}
+```
+* `consignmentId` is the unique identifier for the consignment
+* `date` is the date the error file was created
+* `fileError` is the type of error. SCHEMA_VALIDATION indicates a schema validation error
+* `assetId` for TDR is the file path as this should be unique and the error can be tracked back to the original
```

Comment:
Slight wording change: `... this should be unique within the consignment and the error can be ...`

Reply:
done